src/nyc_trees/apps/survey/admin.py

```diff
@@ -6,6 +6,8 @@
 from django.contrib import admin
 import apps.survey.models as m
 
+from django.contrib.gis.admin.options import GeoModelAdmin
```

Comment:
This unused import is causing the build to fail. Only mentioning it here to speed up feedback since we may (or may not) want this in the next release.